.github/workflows/presubmit-image.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+
+jobs:
+  presubmit-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.20.x'
+      - uses: chainguard-dev/actions/melange-build@main
+        with:
+          config: melange.yaml
+          archs: x86_64
+          sign-with-temporary-key: true
+      - uses: chainguard-images/actions/apko-publish@main
+        with:
+          config: apko.yaml
+          archs: x86_64
+          tag: ttl.sh/kots
```

Comment:
It would probably make integration easier if we match the chainguard image name to our current image name `kotsadm`

Reply:
Additionally, we will usually push the image with a namespace to avoid conflicts on ttl.sh. For example `ttl.sh/<some-identifier/kotsadm:24h`